docs/user-guides/finetuner.md

```diff
@@ -0,0 +1,144 @@
+# Fine-tune Models
+
+Although CLIP-as-service has provided you a list of pre-trained models, you can also train your models. 
+This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fine-tune models and use them in CLIP-as-service.
+
+For installation and basic usage of Finetuner, please refer to [Finetuner website](https://finetuner.jina.ai).
```

Comment:
please redirect to the clip model finetuning tutorial link

Reply:
is this really needed? i think the main page is sufficient, from which users can find useful information. Otherwise we provide a different narrative of the same thing

Reply:
But we need to guild the user to access the clip fineturning tutorial. So far, I cannot find this link in the doc.